packages/create-app/src/run.ts

```diff
@@ -110,6 +110,7 @@ export async function run(
     foundryUrl,
     osdkPackage,
     corsProxy,
+    clientVersion: "^2.0.7",
```

Comment:
Will this run into the same cache issue if we don't update this

Reply:
it will but at least at the point that we wrote this, we know that 2.0.7 should work with the generated code. Otherwise you will see situations where the generated code uses a version of the client that the user doesnt get.